lab-trent/greet.js

```diff
@@ -0,0 +1,7 @@
+module.exports.greet = function(name) {
+  if (process.argv[2] && !name) {
```

Comment:
You're also missing an actual entry point to the application (index.js) even though you assigned the entry point to greet.js in the package.json.

The functionality of grabbing cli arg(s) should be done there in the entry point for this assignment. It should also require in your greet module.